src/main/java/org/ozwillo/dcexporter/model/DcModelMapping.java

```diff
@@ -40,7 +40,8 @@
     private String source;
     private String version;
     private String ckanPackageId;
-    private String ckanResourceId;
+    private String ckanResourceCsvId;
+    private String ckanResourceJsonId;
```

Comment:
Not the cleanest way to implement this.
A array or dictionnary sloud probably be better instead.